src/Components/MoodRating/MoodRating.js

```diff
@@ -127,8 +140,8 @@ const MoodRating = () => {
             {error && <p className={style.errorMessage}>
               <span role='img' alt='Error sign'>
                 ⚠
-              </span>
-              Please select an option above to continue! 
+              </span>{" "}
+              Please select one of the faces above to continue!{" "}
```

Comment:
Nice catch making this error message more specific - I think small changes like that really add to the user experience